packages/code-du-travail-data/indexing/populate.js

```diff
@@ -136,30 +137,30 @@ function* cdtnDocumentsGen() {
   );
 
   logger.info("=== Courriers ===");
-  yield require("../dataset/courrier-type/export-courriers.json").map(
-    ({
-      titre,
-      filename,
-      description,
-      questions,
-      html,
-      tags,
-      date_redaction,
-      redacteur,
-      source
-    }) => ({
-      source: SOURCES.LETTERS,
-      title: titre,
-      slug: slugify(titre),
-      description,
-      text: questions.join("\n"),
-      html,
-      filename,
-      date: date_redaction,
-      editor: source,
-      author: redacteur,
-      tags: flattenTags(tags)
-    })
+  yield getCourriers().then(courriers =>
```

Comment:
yielding a promise unwrap it ? 
o_O  didn't know we could do that :)

Reply:
no you cant :)